national-resources/switzerland.md

```diff
@@ -0,0 +1,79 @@
+
+
+## Health authorities
+<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies --->
+The FOPH plays a critical role in setting national public health goals and guidelines, overseeing the country's health insurance system, and coordinating responses to public health emergencies. It has a Communicable Diseases Division in charge of monitoring, publishing regular reports on the epidemiological situation, laying down and implementing prevention and control strategies. The FOPH also manages national health registries and monitors health indicators, such as disease incidence rates and vaccination coverage. FOPH may also require laboratories, health care institutions and other institutions of the cantons to communicate their stocks of therapeutic products, protective equipment and other medical goods important for the maintenance of health care capacities (important medical goods). [DigiSanté](https://www.bag.admin.ch/bag/fr/home/strategie-und-politik/nationale-gesundheitsstrategien/digisante.html), a national programme to foster the digital transformation of the healthcare systems and public health authorities, has just launched.
+
+In addition to the FOPH, there are several other agencies and institutions involved in shaping public health policy and research in Switzerland. These include a network of National Reference Centers, Swiss societies for microbiology and infectious diseases,  the [Swiss Academy of Medical Sciences](https://www.samw.ch/en.html), the [Swiss Tropical and Public Health Institute](https://www.swisstph.ch/en/about), lobbies and the [Swiss National Science Foundation](https://snf.ch/en), among others.
+
+Switzerland also participates in international public health organisations, such as the World Health Organization (WHO) which provide guidance and support for public health measures at the national and international level.
+
+In March 2020, the Swiss Federal Council declared a state of emergency and announced a series of measures to control the spread of SARS-CoV-2, restricting the cantons’ power of decision. Country's extensive network of laboratories ramped up testing capacity to identify new cases. A national genomic surveillance program was notably implemented by the Reference Laboratory for Emerging Viral Infections in collaboration with the FOPH. The genomic data were generated by the laboratories and fed into the [Swiss Pathogen Surveillance Platform](https://www.spsp.ch) (SPSP),  which sent automatic variants reports to FOPH, allowing it to have and provide regular updates on the variants of concern on the national dashboard.  Together with other contextual data, it supported FOPH in issuing guidance to the public on how to prevent the spread of the virus. The government also launched a contact tracing app to help identify potential COVID-19 cases and prevent further transmission. 
```

Comment:
Add the link to the national dashboard here as well (saw it below as a separate item)